README.md

```diff
@@ -1,123 +1,94 @@
 # Juju Box
 
-Juju Box a docker container project that provides a configured Juju
-environment with all the tools you need to work with Juju.
+Juju Box is a docker container project that provides a configured Juju
+environment with all the tools you need to work with 
+[Juju](https://jujucharms.com).
 
-Juju provisions machines, deploy services and orchestrates them into
-systems.
+Juju is an open source, universal model for applications. Juju allows you to 
+deploy, configure, manage, maintain, and scale cloud services quickly and 
+efficiently on public clouds, as well as on physical servers.
 
-This container comes with the basic tools you need to get started w/
-juju.
+This container comes with the basic tools you need to get started with Juju.
 
  - juju
- - juju quickstart
```

Comment:
No as far as I can tell it is no longer an installable package any longer. I assumed it was removed. If you can find quickstart please let me know.